apps/crn-server/src/handlers/event/helper.ts

```diff
@@ -0,0 +1,12 @@
+import { Payload } from '@asap-hub/algolia';
+import { EventResponse } from '@asap-hub/model';
+
+export const addTagsToEvents = <T extends Payload>(
```

Comment:
Maybe we should turn this into a generic function for the remaining entities that have a `tags` field we are going index later?